lib/vdsm/common/config.py.in

```diff
@@ -455,6 +455,10 @@ parameters = [
             'monitoring. On timeout, the drive will be monitored in the '
             'next monitoring cycle. (default 0.5)'),
 
+        ('extend_timeout', '2.0',
```

Comment:
There are a couple small typos in this commit message that I don't know if they are worth to correct. Feel free to ignore them if you want:

- `This create duplicate extend requests` ⇾ `This creates duplicate extend requests`
- `it skip this drive if a` ⇾ `it skips this drive if a`

Otherwise, all changes look good to me.

Reply:
Sure will fix in next version.